lib/openapi/index.ts

```diff
@@ -0,0 +1,46 @@
+import { createDocument } from "zod-openapi"
+
+import { machinesPaths } from "./machines"
+import { proofsPaths } from "./proofs"
+
+export const document = createDocument({
+  openapi: "3.1.0",
+  info: {
+    title: "Ethproofs API",
```

Comment:
@pettinarip Thoughts on using SITE_NAME from lib/constants wherever we mention the Ethproofs name, to keep it consistent with one canonical reference?

Reply:
I have a separat branch where I was going to update that elsewhere, but was awaiting name confirmation before posting

Reply:
- #116

Reply:
```suggestion
    title: `${SITE_NAME} API`,
```

With import statement: 

```tsx
import { SITE_NAME } from "@/lib/constants"
```